bref

```diff
@@ -248,4 +250,105 @@ $app->command('deployment stack-name [--region=]', function (string $stackName,
     return 0;
 })->descriptions('Displays the latest deployment logs from CloudFormation. Only the logs from the last 24 hours are displayed. Use these logs to debug why a deployment failed.');
 
+$app->command('dashboard [--port=]', function (int $port = 8000, SymfonyStyle $io) {
+    if (! file_exists('serverless.yml')) {
```

Comment:
The `serverless info` at line  289 don't make the job ?

Reply:
In case someone uses SAM, they probably won't be `serverless` installed. So the error message they will get is: "The `serverless` command is not installed…". If they install it it will still not work.

I was thinking it's better to explain clearly that you need to be using `serverless` entirely, not just have it installed.